### PR TITLE
Make the font color of the all the pages as one color

### LINKED ISSUE #1426: Make the font color of the all the scholarx page as one color
**Describe the bug**
Currently, the font color of the description in the scholarx pages are different

**To Reproduce**
1. Go to '[Scholarx 2022](https://sefglobal.org/scholarx/2022/), [Scholarx 2021](https://sefglobal.org/scholarx/past-programs/2021/), [Scholarx 2020](https://sefglobal.org/scholarx/past-programs/2020/), [Scholarx 2019](https://sefglobal.org/scholarx/past-programs/2019/), [Scholarx Archive](https://sefglobal.org/scholarx/archive/)'
2. See error in the description

**Expected behavior**
Make the font color of the all the scholarx page as one color

**Screenshots**

Archive
image

2022
image

**Additional context**
The font color in the 2022, 2021, 2020, 2019 page will affect the footer font color also, therefore remove the black color and make all the pages like archive

Archive footer
image

2022, 2021, 2020, 2019 footers
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1426 

## Goals

Making the font color of the all the scholarx page as one color

## Approach

Made the font color of the all the scholarx page as one color

### Screenshots

image

  
### Preview Link

https://pr-1436-sef-site.surge.sh/scholarx/2022
https://pr-1436-sef-site.surge.sh/scholarx/past-programs/
https://pr-1436-sef-site.surge.sh/scholarx/past-programs/2020/
https://pr-1436-sef-site.surge.sh/scholarx/past-programs/2019/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: scholarx/past-programs/index.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
           href="style.css">
 </head>
 
-<body style="font-family: 'Montserrat', sans-serif; color: black">
+<body style="font-family: 'Montserrat', sans-serif;">
```
Comment (anjula-sack) [inline scholarx/past-programs/index.html]: Let's create a CSS class for this and add this to the global CSS

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: scholarx/past-programs/index.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply scholarx/past-programs/index.html:51]: So I should remove the inline css right? @anjula-sack

Comment (YoshithaRathnayake) [reply scholarx/past-programs/index.html]: Done can you review this now @anjula-sack ?

File: engineering-team.html [hunk 2]
```diff
@@ -31,7 +31,7 @@
         <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
     </head>
 
-<body style="font-family: 'Montserrat', sans-serif; color: black">
```
Comment (YoshithaRathnayake) [inline engineering-team.html:34]: Removed the black font color of the engineering team page. Is it ok?

Comment (anjula-sack) [reply engineering-team.html:34]: yeah

**Review by anjula-sack — APPROVED** [review]
Good job! @YoshithaRathnayake

**Pull request merged**